module2/presentation_inheritance.md

```diff
@@ -169,20 +167,6 @@ Załóżmy, że dodajemy klasę `Screen`. Klasa ta wyświetla na bieżąco inter
 Chcemy też stworzyć klasę reprezentującą ekran dotykowy - `TouchScreen`, który również umożliwia odczyt akcji od użytkownika i ich wyświetlanie.
 <!-- .element: class="fragment fade-in" -->
 
-<style>
-.container {
-    display: flex;
-}
-.col {
-    flex: 1;
-    margin: 1%;
-}
-</style>
-
-<div class="container">
-
-<div class="col">
```

Comment:
Why do you want it to be one column only? :)
It's not visible on GH I guess but it is in a presentation.

Reply:
![pr](https://user-images.githubusercontent.com/65040642/86220059-5175b580-bb83-11ea-938f-1089005bbd04.png)

Reply:
html is visible on https://coders.school/edu/courses/obiektowosc-2-dziedziczenie-interfejsy-klasy-abstrakcyjne/lesson/dziedziczenie/

Reply:
Ok, I haven't seen it. I'll do this in another way. Let me close this PR and deliver changes on my own.